test/copy-filter.js

```diff
@@ -96,6 +96,12 @@ test('ignore: not junk when junk: false', util.testSinglePlatform(async (t, opts
   const targetDir = await copyDirToTempDirWithIgnores(t, opts)
   await assertFileNotIgnored(t, targetDir, 'subfolder/Thumbs.db')
 }))
+test('ignore: package manager lock files', util.testSinglePlatform(async (t, opts) => {
```

Comment:
For consistency with the other default ignore tests:

```suggestion
test('ignore default: package manager lock files', util.testSinglePlatform(async (t, opts) => {
```

Reply:
Renamed